File: truss-compiler/src/main/java/truss/compiler/symbols/PropertyAccessorSymbol.java
package truss.compiler.symbols;

public class PropertyAccessorSymbol extends MethodSymbol {
    @Override
    public SymbolKind getKind() {
        return SymbolKind.PROPERTY_ACCESSOR;
    }

    @Override
    public void accept(SymbolVisitor visitor) {
        if (!visitor.isDone()) {
            visitor.visitPropertyAccessor(this);
        }
    }
}

File: Truss.Compiler/Symbols/PropertyAccessorSymbol.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Truss.Compiler.Symbols {
    public class PropertyAccessorSymbol : MethodSymbol {
        public override SymbolKind Kind {
            get { return SymbolKind.PropertyAccessor; }
        }

        public override void Accept(ISymbolVisitor visitor) {
            if (!visitor.Done) {
                visitor.VisitPropertyAccessor(this);
            }
        }
    }
}
